README.md

```diff
@@ -288,6 +288,31 @@ However for CentOS specifically, you can try the following steps:
  	cd $GOPATH/src/github.com/QuarkChain/goquarkchain/consensus/qkchash/native/ && g++ -shared -o libqkchash.so -fPIC qkchash.cpp -O3 -std=gnu++11
  	cd ../../../cmd/cluster && go build -v
 ``` 
+### Q: Is macOS supported?
```

Comment:
Should we use standalone sections for Mac and CentOS instead of question?  @pingke